include/can/core/messages.hpp

```diff
@@ -720,6 +720,40 @@ struct SetBrushedMotorPwmRequest
         -> bool = default;
 };
 
+struct AddBrushedLinearMoveRequest
+    : BaseMessage<MessageId::add_brushed_linear_move_request> {
+    uint8_t group_id;
```

Comment:
What's the difference between this and a "grip" request? Can't we just combine the two CAN messages since we're really just enhancing the functionality of gripping?

Reply:
the grip request specifies force control, and this specifies position control. they carry similar bits for now, but that won't necessarily always be true - they mean different things